site/_docs/reference.md

```diff
@@ -1483,12 +1483,12 @@ period:
 | {fn UCASE(string)} | Returns a string in which all alphabetic characters in *string* have been converted to upper case
 | {fn REPLACE(string, search, replacement)} | Returns a string in which all the occurrences of *search* in *string* are replaced with *replacement*; if *replacement* is the empty string, the occurrences of *search* are removed
 | {fn ASCII(string)} | Returns the corresponding ASCII code of the first character of *string*; Returns 0 if *string* is empty; Returns NULL if *string* is NULL; Returns the Unicode code point for non-ASCII character
+| {fn LEFT(string, integer)} | Returns the leftmost len characters from the *string*; Returns NULL if *string* is NULL;
+| {fn RIGHT(string, integer)} | Returns the rightmost len characters from the *string*; Returns NULL if *string* is NULL;
```

Comment:
JDBC String function `{fn LEFT(string, integer)` and `{fn RIGHT(string, integer)}`  not create in `SqlJdbcFunctionCall` and `SqlOperatorBaseTest.testJdbcFn()` not test.